transports/janus_rabbitmq.c

```diff
@@ -900,8 +934,31 @@ void *janus_rmq_in_thread(void *data) {
 			if(res == AMQP_STATUS_TIMEOUT || res == AMQP_STATUS_SSL_ERROR)
 				continue;
 			JANUS_LOG(LOG_VERB, "Error on amqp_simple_wait_frame_noblock: %d (%s)\n", res, amqp_error_string2(res));
-			break;
+
+			rmq_client->connected = 0;
+
+			/* Try and reconnect */
+			if(rmq_client->rmq_conn && rmq_client->rmq_channel) {
```

Comment:
I don't think that checking `rmq_channel` is needed here

Reply:
👍